spec/services/activitypub/process_status_update_service_spec.rb

```diff
@@ -46,6 +46,29 @@
       expect(status.reload.spoiler_text).to eq 'Show more'
     end
 
+    context 'with no changes at all' do
+      let(:payload) do
+        {
+          '@context': 'https://www.w3.org/ns/activitystreams',
+          id: 'foo',
+          type: 'Note',
+          content: 'Hello world',
+        }
+      end
+
+      before do
+        subject.call(status, json)
+      end
+
+      it 'does not create any edits' do
+        expect(status.reload.edits).to be_empty
+      end
```

Comment:
I'm not sure why the test passes, since I understand `create_previous_edit!` to be unconditional…

Reply:
Yeah, the test does not pass on my computer. It is a mystery how it passed on CI.

Reply:
Not the kind of mystery I enjoy :grimacing: ; anyway the test is expected to pass, now…